extensions/msal-node-extensions/CHANGELOG.json

```diff
@@ -1,6 +1,21 @@
 {
   "name": "@azure/msal-node-extensions",
   "entries": [
+    {
+      "date": "Wed, 29 Sep 2021 17:53:33 GMT",
+      "tag": "@azure/msal-node-extensions_v1.0.0",
+      "version": "1.0.0",
+      "comments": {
+        "patch": [
```

Comment:
This change should be marked as prerelease, instead of patch. Fixing the relevant change file should fix.